src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/UPEPageSteps.scala

```diff
@@ -66,68 +67,69 @@ class UPEPageSteps extends CommonFunctions {
 
       case "UPE Person/Team name" =>
         Wait.waitForTagNameToBeRefreshed("h1")
-        Wait.waitForElementToPresentById(UPEContactNamePage.contactName)
-        Input.sendKeysById(UPEContactNamePage.contactName,name)
+        Input.sendKeysById(UPEContactNamePage.contactName, name)
+        clickByCss(InputUPENamePage.continue)
+
 
       case "UPE Email address" =>
-        Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentById(UPEContactEmailPage.emailField)
         Input.sendKeysById(UPEContactEmailPage.emailField, name)
+        clickByCss(InputUPENamePage.continue)
 
       case "Telephone Number" =>
-        Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(InputUPETelephonePage.telephoneNumber)
         Input.sendKeysByCss(name, InputUPETelephonePage.telephoneNumber)
+        clickByCss(InputUPENamePage.continue)
 
       case "NFM Contact name" =>
-        Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(NFMContactNamePage.contactName)
         Input.sendKeysByCss(name, NFMContactNamePage.contactName)
+        clickByCss(InputUPENamePage.continue)
 
       case "NFM Contact Email" =>
-        Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(NFMContactEmailPage.contactEmail)
         Input.sendKeysByCss(name, NFMContactEmailPage.contactEmail)
+        clickByCss(InputUPENamePage.continue)
 
       case "NFM Telephone Number" =>
-        Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(InputNFMTelephonePage.telephoneNumber)
         Input.sendKeysByCss(name, InputNFMTelephonePage.telephoneNumber)
+        clickByCss(InputUPENamePage.continue)
 
       case "Contact Details Name" =>
-        Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(ContactDetailsInputNamePage.contactName)
         Input.sendKeysByCss(name, ContactDetailsInputNamePage.contactName)
+        clickByCss(InputUPENamePage.continue)
 
       case "Contact Name" =>
-        Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(ContactDetailsInputNamePage.contactName)
         Input.sendKeysByCss(name, ContactDetailsInputNamePage.contactName)
+        clickByCss(InputUPENamePage.continue)
 
       case "Contact Email" =>
-        Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(ContactDetailsInputEmailPage.contactEmail)
         Input.sendKeysByCss(name, ContactDetailsInputEmailPage.contactEmail)
+        clickByCss(InputUPENamePage.continue)
 
       case "Contact Telephone" =>
-        Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(ContactDetailsInputTelephonePage.contactTelephone)
         Input.sendKeysByCss(name, ContactDetailsInputTelephonePage.contactTelephone)
+        clickByCss(InputUPENamePage.continue)
```

Comment:
same as above, we can keep this code after switch block

Reply:
Continue button cannot be used for Address page fields (Address line 1, City, Postal, Country) as many field values need to be added in Address page.